bmp-string.go

```diff
@@ -2,9 +2,10 @@ package pkcs12
 
 import (
 	"unicode/utf16"
+	"unicode/utf8"
 )
 
-func bmpString(s string) ([]byte, error) {
+func bmpString(utf8String []byte) ([]byte, error) {
```

Comment:
One advantage of []rune is that the type system ensures the caller to ConvertToPEM doesn't mix up pfxData and utf8Password (`pfxData []byte, utf8Password []rune`).

But it probably makes sense to stick with what DecryptPEMBlock does.